app/domain/output_document.rb

```diff
@@ -21,7 +21,7 @@ def attendee_name
   end
 
   def appointment_date
-    appointment_summary.date_of_appointment.to_s(:long)
+    appointment_summary.date_of_appointment.strftime('%e %B %Y')
```

Comment:
Maybe http://guides.rubyonrails.org/i18n.html#adding-date-time-formats






On Monday, 30 Mar 2015 at 17:12, Steven Wilkin <notifications@github.com>, wrote:
Message contents can't be shown right now